sdk/keyvault/keyvault/src/az_keyvault_client.c

```diff
@@ -16,11 +16,28 @@
  * {query}                          = 70               ** 200 **
  */
 enum { MAX_URL_SIZE = 200 };
+enum { MAX_BODY_SIZE = 1024 };
 
 static az_span const AZ_KEY_VAULT_KEY_TYPE_KEY_STR = AZ_CONST_STR("keys");
 static az_span const AZ_KEY_VAULT_KEY_TYPE_SECRET_STR = AZ_CONST_STR("secrets");
 static az_span const AZ_KEY_VAULT_KEY_TYPE_CERTIFICATE_STR = AZ_CONST_STR("certificates");
 
+static az_span const AZ_KEY_VAULT_WEB_KEY_TYPE_EC_STR = AZ_CONST_STR("EC");
+static az_span const AZ_KEY_VAULT_WEB_KEY_TYPE_EC_HSM_STR = AZ_CONST_STR("EC-HSM");
+static az_span const AZ_KEY_VAULT_WEB_KEY_TYPE_RSA_STR = AZ_CONST_STR("RSA");
+static az_span const AZ_KEY_VAULT_WEB_KEY_TYPE_RSA_HSM_STR = AZ_CONST_STR("RSA-HSM");
+static az_span const AZ_KEY_VAULT_WEB_KEY_TYPE_OCT_STR = AZ_CONST_STR("oct");
+
+static az_span const AZ_KEY_VAULT_CREATE_KEY_URL_KEYS = AZ_CONST_STR("keys");
+static az_span const AZ_KEY_VAULT_CREATE_KEY_URL_CREATE = AZ_CONST_STR("create");
+static az_span const AZ_KEY_VAULT_CREATE_KEY_BODY_START = AZ_CONST_STR("{ \"kty\": ");
+static az_span const AZ_KEY_VAULT_CREATE_KEY_BODY_END = AZ_CONST_STR(" }");
+
+static az_span const AZ_HTTP_REQUEST_BUILDER_HEADER_CONTENT_TYPE_LABEL
+    = AZ_CONST_STR("Content-Type");
+static az_span const AZ_HTTP_REQUEST_BUILDER_HEADER_CONTENT_TYPE_JSON
```

Comment:
Could we move these generic HTTP header fields into one of policies?

Reply:
Not sure.
Do you mean the logic of adding content-type?  or just the reference the static strings?
Not every http request need a content-type, or can use default `encoded_form` 
right now, only key_vault is using this, 
probably in the future it will make sense to move this else where to reuse it. but for now this is client specific for key vault